localstack/config.py

```diff
@@ -1051,6 +1051,31 @@ def legacy_fallback(envar_name: str, default: T) -> T:
 # e.g. CFN_RESOURCE_PROVIDER_OVERRIDES='{"AWS::Lambda::Version": "GenericBaseModel","AWS::Lambda::Function": "ResourceProvider"}'
 CFN_RESOURCE_PROVIDER_OVERRIDES = os.environ.get("CFN_RESOURCE_PROVIDER_OVERRIDES", "{}")
 
+# bind address of local DNS server
+DNS_ADDRESS = os.environ.get("DNS_ADDRESS") or "0.0.0.0"
```

Comment:
I wonder if we should not default to 127.0.0.1 here. So we do not expose it to the whole network without need.

Reply:
Makes sense. We already default `GATEWAY_LISTEN` to `127.0.0.1` if not in docker and `0.0.0.0` otherwise (using `localstack.config.default_ip`). I can implement that.

Reply:
Then we could also skip the override in dns_server.py, which would be really nice!